frontend/src/actions/countdownActions.js

```diff
@@ -0,0 +1,11 @@
+//@flow
+
+export const startCountdown = (minutes: number) => async (dispatch: *) => {
```

Comment:
`reducers/exercise/index.js` already keeps track of a timer. You can modify that reducer to include the countdown in minutes and an actual start time if you like but we should keep it one place.